app/views/jobseekers/job_applications/build/ask_for_support.html.slim

```diff
@@ -28,7 +28,7 @@
           = f.govuk_text_area :support_needed_details, rows: 10
         = f.govuk_radio_button :support_needed, :no
 
-      = f.govuk_submit current_jobseeker.job_applications.not_draft.none? ? t("buttons.save_and_continue") : t("buttons.save_changes") do
+      = f.govuk_submit redirect_to_review? ? t("buttons.save") : t("buttons.save_and_continue") do
```

Comment:
this condition is repeated a lot, can it be put in helper method?

Reply:
Unless we decide 

> Ternary operators are evil!!!

I'd be a bit averse to adding a helper method, just because I don't think it would make much difference either way. 

But if we do want to decide that as a team then I am really happy to do it!

Reply:
ternary operators are fine by me (unless people nest them, that is evil. and not what you are doing here). but you have 5+ instances of the same ternary operator and if they were in one place that is one peice of code to potentially maintain and im thinking DRY. its no biggie. just an observation and trying to feedback on PR

Reply:
i like ternary operators, i think they read well.

Reply:
Are you thinking something like...

```suggestion
= f.govuk_submit some_button_text_helper do
```

Reply:
pretty much

Reply:
as much as i think ternary operators are far nicer than a 5+ line if/else, a well named method is even better

Reply:
Let me know what you think.

I'm generally a bit wary of helper methods - particularly for something this simple. But I'm happy to go with this if you prefer

Reply:
horses for courses, i prefer it